### PR TITLE
Document Search Sorts by Date Added then Alphabetical 

### PR DESCRIPTION
**Pull request by CEBergin**
updated search to sort by date and display name for documents as per … EE-664

Added a check to see if the schema was a document and then used a different sort.
On both public project documents and admin documents.

### CODE REVIEW

**Comment by a1astair** [comment]
@CEBergin could we get a unit test for this?

File: api/controllers/search.js [hunk 1]
```diff
@@ -454,20 +455,47 @@ var searchCollection = async function (roles, keywords, schemaName, pageNum, pag
...
 
+
+  
   console.log("modifier:", modifier);
   console.log("match:", match);
 
-  var sortingValue = {};
-  sortingValue[sortField] = sortDirection;
+
+  if (schemaName == "Document"){
```
Comment (KitArmstrong) [inline api/controllers/search.js]: This is a great start, but I don't think it is going to keep the same sorting behaviour we currently have. By default the documents should sort by the date and display name; however, the user has the ability to click on a header which would indicate they only want to sort by a single column.

File: api/controllers/search.js [hunk 2]
```diff
@@ -454,20 +455,47 @@ var searchCollection = async function (roles, keywords, schemaName, pageNum, pag
...
+    twoSorts = true;
+  }else {
+    sortingValue = {};
```
Comment (KitArmstrong) [inline api/controllers/search.js]: A good rule of thumb to try and avoid modifying any parameters. I might try creating a new variable to handle these values.

**Comment by CEBergin** [comment]
@a1astair  I'm not sure a unit test would be practical considering searchCollection is 400 lines that we're hoping to refactor soon. May be worth waiting until the refactor to avoid duplication of effort.

**Pull request merged**